src/admin/pages/ProductCreate.js

```diff
@@ -3,59 +3,82 @@ import { addNewProduct } from "../../store/actions";
 import { Form, Button, Row, Col } from 'react-bootstrap';
 import { useForm } from "react-hook-form";
 import { getCategories } from "../../store/selectors";
+import { useHistory, useParams } from "react-router";
+import { getProducts } from "../../store/selectors/products.selectors";
+import { useEffect } from "react";
+import { editProduct } from "../../store/actions";
 
-export const ProductCreate = () => {
+export const ProductCreate = ({ type }) => {
```

Comment:
This name should be changed into something like Product (Page) or ProductPreview since this component is being used for preview/edit in addition to creating.